docs/config/cluster.rst

```diff
@@ -1150,6 +1150,7 @@ a node. There are several pools, but the important ones include:
 * ``search``: For count/search operations, defaults to fixed
 * ``get``: For queries on ``sys.shards`` and ``sys.nodes``, defaults to fixed.
 * ``refresh``: For refresh operations, defaults to cache
+* ``generic``: For internal management task like updating/writing the cluster state, fetching internal information, leader/follower checks, etc.
```

Comment:
Maybe it's better to be more generic here like:

```suggestion
* ``generic``: For internal management tasks like cluster management, snapshot management, etc.
```
And I think the leader/follower stuff falls under the next thread pool.

Reply:
Maybe there are too many "management" in that definition?

Reply:
maybe then `For internal tasks like: ....`
@romseygeek maybe you have better phrasing suggestions?

Reply:
`For internal tasks such as cluster and snapshot management` reads fine to me.

Reply:
Sorry I couldn't help much with a better definition. I just added what Basti shared with me when asked about the generic thread pool.

Reply:
No worries @karynzv! Thx for spotting it, and helping improve our docs!